### DISCUSSION TITLE
Idea: How to create a plugin on kaboom

### DISCUSSION THREAD

**Discussion by Stutz** [replit/kaboom#150] [Ideas]
Hello,

I suggest to create a documentation "how to build plugins for the kaboom library" on the site. 
This would allow other people to build and contribute with new features to kaboom.

**Comment by lajbel** [comment]
at the end of the official website there is something about that!

**Reply by pelikhan** [reply]
@lajbel do you have a link? Maybe it moved off but I can't find it on https://kaboomjs.com/

**Reply by lajbel** [reply]
https://kaboomjs.com/doc/comp :)

**Comment by Stutz** [comment]
OMG, I didn't see it! 
There aren't links to "Plugin system" neither to "Custom component" on the website menu. 
They're "hidden" on the bottom site.
In this case, I suggest to put links to them on the menu.

**Reply by lajbel** [reply]
It would be a nice addition @slmjkdbtl

**Reply by slmjkdbtl** [reply]
Yep! Will do

**Comment by pelikhan** [comment]
Thanks!
________________________________
